src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -585,6 +607,7 @@ private bool ReadDelimited(bool waitChanges, ArrayList blocks, bool readBackward
             Span<char> readBuffer = stackalloc char[currentOffset];
             bool delimiterNotFound = true;
             _currentLineContent.Clear();
+            char currentChar;
```

Comment:
`currentChar` can be defined below where it is used.

Reply:
Fixed.